test/integration/gnmiutils.go

```diff
@@ -74,50 +75,50 @@ func GNMIGet(ctx context.Context, c client.Impl, paths []DevicePath) ([]DevicePa
 	getTZRequest := &gpb.GetRequest{}
 	if err := proto.UnmarshalText(protoString, getTZRequest); err != nil {
 		fmt.Printf("unable to parse gnmi.GetRequest from %q : %v", protoString, err)
-		return nil, err
+		return nil, nil, err
 	}
 
 	response, err := c.(*gclient.Client).Get(ctx, getTZRequest)
 	if err != nil || response == nil {
-		return nil, err
+		return nil, nil, err
 	}
-
-	return convertGetResults(response)
+	pathsResp, errResp := convertGetResults(response)
```

Comment:
Maybe the convertGetResults() should be extracting the extensions from the result too?

Reply:
yeah, better approach, changed